packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -186,13 +193,28 @@ function NavigationMenuItemEdit( {
 					'is-selected': isSelected,
 				} ) }
 			>
-				<RichText
-					className="wp-block-navigation-menu-item__content"
-					value={ label }
-					onChange={ ( labelValue ) => setAttributes( { label: labelValue } ) }
-					placeholder={ __( 'Add item…' ) }
-					withoutInteractiveFormatting
-				/>
+				<div className="wp-block-navigation-menu-item__inner">
+					<RichText
+						className="wp-block-navigation-menu-item__content"
+						value={ label }
+						onChange={ ( labelValue ) => setAttributes( { label: labelValue } ) }
+						placeholder={ itemLabelPlaceholder }
+						withoutInteractiveFormatting
+					/>
+
+					{ isLinkOpen &&
+						<LinkControl
+							className="wp-block-navigation-menu-item__inline-link-input"
+							onKeyDown={ handleLinkControlOnKeyDown }
+							onKeyPress={ ( event ) => event.stopPropagation() }
+							currentLink={ link }
+							onLinkChange={ updateLink( setAttributes, label ) }
+							onClose={ () => setIsLinkOpen( false ) }
+							currentSettings={ { 'new-tab': opensInNewTab } }
```

Comment:
This PR will cause this to break 

https://github.com/WordPress/gutenberg/pull/18285

Reply:
No worries, it just happens. Let's see which ships first and then let's simply update the looser :-D